Emulator/JIT/Generic/TJITGenericROMPatch.cpp

```diff
@@ -73,6 +73,34 @@ TJITGenericPatch gNewtConfigPatch(0x000013fc, kROMPatchVoid, kROMPatchVoid, kROM
 								  | 0x00008000 /*kEnableStdout*/,
 								  "gNewtConfig patch");
 
+/*
+ * Intercept DebugStr calls and prints them to stdout. stdout is chosen so that
+ * Einstein does not need to be configured for DEBUG builds, nor is a logger instance needed.
```

Comment:
Shouldn't we use a logger but not KPrintf here?
Indeed, I believe the logger is unrelated to DEBUG build except that KPrintf was introduced to restrict logging to DEBUG builds. And in this very case, we probably want to capture the output of DebugStr calls to present output to user in a distinct window.

Reply:
Right, so would this best be the Emulator logger? In any case, `TARMProcessor` doesn't have an own logger instance (and I think it wouldn't need one), so we'd need to get a logger from somewhere. Maybe expose the Emulator logger?

Reply:
Indeed, it makes sense!

Reply:
I am confused as TARMProcessor does have a logger.

Reply:
Oh right, there it is, I guess it would need an accessor (or make the `TGenericPatch*` classes `friend` classes?)

Reply:
Or add a new method to TEmulator to log a debug string.
I don't have a strong opinion on the best way to do this.

Reply:
I exposed the logger now, it was a bit simpler.